website/docs/addon_resolve_admin.md

```diff
@@ -14,11 +14,17 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+:::caution Free Resolve limitation
+
+Free Resolve won't work properly with our resolve addon.
+This limitation exists due to the constraints set by BMD Resolve and not our own. It's important to note here that the free version of resolve does not grant access to Python API.
+:::
```

Comment:
```suggestion
:::caution Requires Resolve Studio

The free edition of Resolve won't work properly with the AYON resolve addon.
This is because the Resolve Studio is required for external Python API access needed for the integration.
:::
```
I wonder if this is worded better.

We might also want to avoid naming it "our addon" but just the "AYON addon" or alike?

Reply:
I find it more accurate.. especially `external Python API access`  which quite descriptive and would have saved me that google search (in which I found that free resolve supports some python scripting!)

what do you think @jakubjezek001 ?